### PR TITLE
Create 06-transferring-props.ru-RU.md

### PR DESCRIPTION
**Pull request by alexzherdev**
cc @gaearon

### CODE REVIEW

File: docs/docs/06-transferring-props.ru-RU.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+---
+id: transferring-props-ru-RU
+title: Передача свойств
```
Comment (alexzherdev) [inline docs/docs/06-transferring-props.ru-RU.md]: "Передача" не совсем отражает суть слова в данном случае, это больше похоже на "пробрасывание"? Но не уверен, что такое слово можно использовать.

File: docs/docs/06-transferring-props.ru-RU.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+);
+```
+
+Но что насчет свойства `name`? Или свойства `title`? Или `onMouseOver`?
+
+## Передача с помощью `...` в JSX
+
+> ЗАМЕЧАНИЕ:
+>
+> Синтаксис `...` является частью предложения об оставшихся свойствах и расширении свойств объектов. Это предложение на пути к тому, чтобы стать стандартом. Обратитесь к главе [Оставшиеся свойства и расширение свойств ...](/react/docs/transferring-props-ru-RU.html#-----...), чтобы узнать об этом подробнее.
```
Comment (alexzherdev) [inline docs/docs/06-transferring-props.ru-RU.md]: `transferring-props-ru-RU.html#-----...` - yuck. This works, but illustrates how jekyll (or markdown?) fails at generating readable section links with any language other than English.

File: docs/docs/06-transferring-props.ru-RU.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
+---
+id: transferring-props-ru-RU
+title: Передача свойств
+permalink: transferring-props-ru-RU.html
+prev: reusable-components-ru-RU.html
```
Comment (alexzherdev) [inline docs/docs/06-transferring-props.ru-RU.md]: Depends on #7234

File: docs/docs/06-transferring-props.ru-RU.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+В React распространен шаблон, заключающийся в построении абстракции вокруг компонента. Внешний компонент выставляет единственное свойство для достижения чего-то, что, возможно, имеет сложную внутреннюю реализацию.
+
+Вы можете использовать [расширение атрибутов JSX](/react/docs/jsx-spread-ru-RU.html), чтобы объединить старые свойства с дополнительными значениями:
+
+```javascript
+<Component {...this.props} more="values" />
+```
+
+Если вы не пользуетесь JSX, вы можете пользоваться любыми вспомогательными методами, такими как `Object.assign` в ES6 или `_.extend` из Underscore:
```
Comment (nkt) [inline docs/docs/06-transferring-props.ru-RU.md]: Если вы используете JSX, вы можете использовать любыми функциями языка, такими как `Object.assign` в ES2015 или `_.extend` из Underscore:

File: docs/docs/06-transferring-props.ru-RU.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+```javascript
+React.createElement(Component, Object.assign({}, this.props, { more: 'values' }));
+```
+
+Оставшаяся часть этой статьи объясняет лучшие практики. Здесь используется JSX и экспериментальный синтаксис ECMAScript.
```
Comment (nkt) [inline docs/docs/06-transferring-props.ru-RU.md]: Далее мы рассмотрим лучшие практики. ...

File: docs/docs/06-transferring-props.ru-RU.md (hunk 2 above)
Comment (zpao) [inline docs/docs/06-transferring-props.ru-RU.md]: FWIW, we're actually trying to do this ourselves (https://github.com/facebook/react/blob/master/docs/_plugins/header_links.rb). We can probably make it better!

Comment (alexzherdev) [inline docs/docs/06-transferring-props.ru-RU.md]: @zpao Ah, didn't know about that, sorry 😊 I have a fix for this to include Asian languages and Russian, but will need to update the links in existing docs. I can put up a PR when that's ready.

Comment (alexzherdev) [inline docs/docs/06-transferring-props.ru-RU.md]: Trying to fix this in #7497

File: docs/docs/06-transferring-props.ru-RU.md (hunk 4 above)
Comment (alexzherdev) [inline docs/docs/06-transferring-props.ru-RU.md]: Там "If you don't use JSX", так что тогда "не используете JSX".
Не совсем согласен с "функции языка": `Object.assign` хоть и можно назвать таковой, но `_.extend` не подходит под определение "функции языка".
Кроме того, получается ИМХО далековато от оригинальной фразы "object helper".

**Comment by gaearon** [comment]
Thanks for the PR!

We just released [new docs mostly written from scratch](https://facebook.github.io/react/). However this means that all old localized content is gone, as new one needs to be written.

We will track this effort in https://github.com/facebook/react/issues/8063. The goal is to make it easier for people to review localizations so you wouldn't have to wait for the core team, and your PRs would be reviewed much faster.

Sorry, and thank you for your work!

**Pull request closed without merging**